test/webpack.config.js

```diff
@@ -36,8 +36,13 @@ module.exports = async (env, options) => {
                 {
                     test: /\.ts$/,
                     exclude: /node_modules/,
-                    use: "babel-loader"
-                },  
+                    use: {
+                      loader: "babel-loader",
+                      options: {
+                        presets: ['@babel/preset-typescript']
```

Comment:
This should be @babel/present-env

Reply:
The js entry uses the env case . . . this is the .tx$ entry.  I don't know why it's there, but it wasn't something I was going to track down for this change.

Reply:
OK, the tests are using TypeScript and this is the separate tests/webpack.config.js file.  I guess this is OK